samplers/jaegerremote/sampler_remote.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/gogo/protobuf/jsonpb"
```

Comment:
```suggestion
	"github.com/gogo/protobuf/jsonpb"
	
```

Reply:
Because the generate uses gogoproto, Protobuf V2 cannot be used